frontend/src/old-pages/Clusters/JobSubmitDialog.tsx

```diff
@@ -9,6 +9,7 @@
 // OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 // limitations under the License.
 import * as React from 'react';
+import {Trans, useTranslation} from 'react-i18next';
```

Comment:
Is _Trans_ strictly needed?

Reply:
Maybe not, I'll do a test using only `useTranslation` and eventually send a new revision.

Reply:
Verified that we can do everything with the function, changed in rev2.